RecoLocalTracker/SiPixelClusterizer/plugins/alpaka/CalibPixel.h

```diff
@@ -0,0 +1,134 @@
+#ifndef RecoLocalTracker_SiPixelClusterizer_plugins_alpaka_CalibPixel_h
+#define RecoLocalTracker_SiPixelClusterizer_plugins_alpaka_CalibPixel_h
+
+#include <algorithm>
+#include <cstdint>
+#include <cstdio>
+#include <type_traits>
+
+#include <alpaka/alpaka.hpp>
+
+#include "CondFormats/SiPixelObjects/interface/SiPixelGainCalibrationForHLTLayout.h"
+#include "CondFormats/SiPixelObjects/interface/alpaka/SiPixelGainCalibrationForHLTDevice.h"
+#include "CondFormats/SiPixelObjects/interface/alpaka/SiPixelGainCalibrationForHLTUtilities.h"
+#include "DataFormats/SiPixelClusterSoA/interface/ClusteringConstants.h"
+#include "DataFormats/SiPixelClusterSoA/interface/SiPixelClustersSoA.h"
+#include "DataFormats/SiPixelDigiSoA/interface/SiPixelDigiErrorsSoA.h"
+#include "DataFormats/SiPixelDigiSoA/interface/SiPixelDigisSoA.h"
+#include "Geometry/CommonTopologies/interface/SimplePixelTopology.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/traits.h"
+#include "RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelClusterThresholds.h"
```

Comment:
Strictly speaking this `#include` is not allowed. I'd suggest to move the `SiPixelClusterThresholds.h` to `RecoLocalTracker/SiPixelClusterizer/interface`, and maybe add a comment that it is an implementation detail of the package.

Reply:
Done.